### PR TITLE
Enable egg installation on Python 3

### PR DESCRIPTION
**Pull request by cillianderoiste**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -2,12 +2,19 @@ Changelog
...
 ------------------
 
 - Nothing changed yet.
 
 
+1.1.0 (2023-02-14)
+------------------
+
+- Remove the requirement for Python 2.7
+  [cillianderoiste]
```
Comment (gforcada) [inline CHANGES.rst:9]: You probably want to move that under the `(unreleased)` header.

File: CHANGES.rst [hunk 2]
```diff
@@ -2,12 +2,19 @@ Changelog
...
-1.0.1 (unreleased)
+1.1.1 (unreleased)
 ------------------
 
 - Nothing changed yet.
 
 
+1.1.0 (2023-02-14)
+------------------
+
```
Comment (gforcada) [inline CHANGES.rst]: Depending on the tooling you use to create a new release, this is done automatically, `zest.releaser` does that for example.

I see on [this add-on's pypi's page](https://pypi.org/project/collective.splitsitemap/#history) that this release has not been made and only @frapell can actually publish new releases

**Review by gforcada — CHANGES_REQUESTED** [review]

File: CHANGES.rst (hunk 2 above)
Comment (frapell) [reply CHANGES.rst]: @gforcada is your pypi user `gforcada` as well? I can give you access if you want

Comment (gforcada) [reply CHANGES.rst]: no, is `gilforcada` but I don't have much interest in this package, I reviewed the PR only 😓 you might give access to the `collective` user that exists on pypi to help manage releases of packages though 👍🏾

Comment (frapell) [reply CHANGES.rst]: done! :)

**Comment by mauritsvanrees** [comment]
I have made a release: https://pypi.org/project/collective.splitsitemap/1.1.0/

**Pull request merged**